.github/workflows/misc_checks.yaml

```diff
@@ -0,0 +1,24 @@
+name: Misc. Checks
```

Comment:
Can we have something more descriptive here? What about simply "Code style" or "Code style and compliance"? Those examples seem to broadly cover what's done.